google/ggrc-core
# [5Obj] GGRC-8504 Add “migrationDate” attribute to models

# Comment

**Should be merged to "GGRC-7200".**

# Issue description

Add “migrationDate” attribute to 5 models.

# Steps to test the changes

1. Open Change Log tab for each model.
2. Check date near "Open Change Log in new frontend" button.
          
# Solution description
Add “migrationDate” attribute to 5 models.


# Sanity checklist




# PR Review checklist
